app/views/achievements/index.html.haml

```diff
@@ -1,4 +1,4 @@
 .row
-  .col-xs-12
+  .col-md-12
```

Comment:
I believe this change is redundant. The grid classes indicate what size they should be used at. `col-xs-12` means it should be full width even at the smallest screen size. (Compare to `col-md-3` which means it should be 4 across from medium sizes; at smaller resolutions it will become full width (try resizing the browser to see.)